cernopendata/modules/pages/views.py

```diff
@@ -181,19 +181,23 @@ def about_cms_dataset_names():
 @blueprint.route('/getting-started/<exp>')
 def getting_started_redirect(exp):
     """Redirects to associated experiment."""
-    return redirect('/docs/%s-getting-started' % exp,
-                    code=302)
+    if exp == "cms":
+        return redirect('/docs/cms-getting-started-2011')
+    return redirect('/docs/%s-getting-started' % exp)
```

Comment:
Works nicely for the root of the problem.  Could you also look at sanitising the arguments? E.g. `year` is a free text field and users can pass anything there... E.g. try for example `firefox http://0.0.0.0:5000/vm/cms/../alice`.  It would be nice to sanitise both `exp` and `year` to consolidate this. (as part of this PR or another later PR)